IEnvoyProxy/go.mod

```diff
@@ -1,19 +1,164 @@
 module IEnvoyProxy
 
-go 1.16
-
-replace (
-	git.torproject.org/pluggable-transports/snowflake.git/v2 => ../snowflake
-	github.com/lucas-clemente/quic-go => github.com/tobyxdd/quic-go v0.27.1-0.20220516000630-9265b64059b0
-	github.com/pion/dtls/v2 => github.com/pion/dtls/v2 v2.0.12
-	github.com/tobyxdd/hysteria v1.0.5 => ../hysteria
-	github.com/v2fly/v2ray-core v4.45.2+incompatible => ../v2ray-core
-)
+go 1.19
 
 require (
 	git.torproject.org/pluggable-transports/snowflake.git/v2 v2.0.0-00010101000000-000000000000
-	github.com/tobyxdd/hysteria v1.0.5
+	github.com/apernet/hysteria/app v1.3.5
 	github.com/v2fly/v2ray-core v4.45.2+incompatible
-	github.com/v2fly/v2ray-core/v5 v5.0.7 // indirect
-	golang.org/x/mobile v0.0.0-20230427221453-e8d11dd0ba41 // indirect
+)
+
+require (
```

Comment:
that's quite a require list there o_O

Reply:
Yeah, Hysteria added a bunch of dependencies: https://github.com/apernet/hysteria/blob/master/app/go.mod